product_variant_configurator/models/product_configurator_attribute.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# © 2015 Oihane Crucelaegui - AvanzOSC
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2016 ACSONE SA/NV
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3
+
+from openerp import api, fields, models
+from openerp.addons import decimal_precision as dp
+
+
+class ProductConfiguratorAttribute(models.Model):
+    _name = 'product.configurator.attribute'
+
+    owner_model = fields.Char(required=True)
+
+    owner_id = fields.Integer(string="Owner",
+                              required=True,
+                              ondelete="cascade")
```

Comment:
@zakiuu I understand why it is an Integer. What I don't understand is the ondelete=cascade. It has no effect, IMO.